skyrim-platform/src/platform_se/CMakeLists.txt

```diff
@@ -24,7 +23,7 @@ if(MSVC)
   target_link_libraries(cef INTERFACE "${CEF_SHARE}/$<CONFIG>/libcef.lib")
   target_include_directories(cef INTERFACE ${CEF_INCLUDE_DIR})
 
-  set(third_party "${SKYRIM_PLATFORM_SOURCE_DIR}/src/platform_se/third_party")
+  set(third_party "${SKYRIM_PLATFORM_SOURCE_DIR}/src/third_party")
```

Comment:
What's the purpose of this?

Reply:
This is now used in `platform_lib` too. `platform_lib` is a new target in skyrim-platform that can be used as a library without Skyrim SE. This is useful for unit testing. This PR places these three directories (`platform_se`, `platform_lib`, `third_party`) into `src`